test/webbrowser.js

```diff
@@ -141,28 +140,41 @@ WebBrowser.prototype = {
     var cmdKillAll, cmdCheckAllKilled, isAllKilled;
 
     if (process.platform === 'win32') {
-      var wmicPrefix = 'wmic process where "not Name = \'cmd.exe\' ' +
+      var wmicPrefix = ['process', 'where', '"not Name = \'cmd.exe\' ' +
         'and not Name like \'%wmic%\' ' +
-        'and CommandLine like \'%' + this.uniqStringId + '%\'" ';
-      cmdKillAll = wmicPrefix + 'call terminate';
-      cmdCheckAllKilled = wmicPrefix + 'get CommandLine';
+        'and CommandLine like \'%' + this.uniqStringId + '%\'"'];
+      cmdKillAll = {
+        file: 'wmic',
+        args: wmicPrefix.concat(['call', 'terminate'])
+      };
+      cmdCheckAllKilled = {
+        file: 'wmic',
+        args: wmicPrefix.concat(['call', 'get CommandLine'])
```

Comment:
I think `'call'` should be removed here, i.e., it should probably be `wmicPrefix.concat(['get', 'CommandLine'])` like the other one.